codec/aminojson/testdata/legacyamino.json

```diff
@@ -0,0 +1 @@
+{"message":{"foo":"test"},"enum":1,"str_map":{"foo":"abc"},"repeated":[3,-7,2,6,4],"str":"abcxyz\"foo\"def","bool":true,"bytes":"AAECAw==","i32":-15,"f32":1001,"u32":1200,"si32":-376,"sf32":-1000,"i64":"14578294827584932","f64":"9572348124213523654","u64":"4759492485","si64":"-59268425823934","sf64":"-659101379604211154","any":{"type_url":"type.googleapis.com/testpb.NestedMessage","value":"Cg9hbnkgdHlwZSBuZXN0ZWQQCw=="},"timestamp":{"seconds":"1641040260"},"duration":{"seconds":"3000"}}
```

Comment:
How is this file (as well as stablejson.json) used?

Reply:
It's not.. deleted.